### DISCUSSION TITLE
插件中的笔记文件存储的本地路径是什么？

### DISCUSSION THREAD

**Discussion by tyehuoPHD** [windingwind/zotero-better-notes#382] [Q&A] [answered]
# 插件中的笔记文件存储的本地路径是什么？
我不希望莫名其妙的损失掉笔记。

**Comment by LZleejean** [comment]
同文，猜测是zotero内部

**Reply by tyehuoPHD** [reply, marked as answer]
在zotero的数据文件夹中的*.sqlite数据库文件中。*.sqlite非常重要，里面包含各种note，文本什么的，但是不包含pdf。笔记并不以单独文件存在。

**Reply by Zhihua-Lee** [reply]
请问一下，您知道better notes里，笔记的插图是保存在哪里吗？

我在网页版zotero中的web libraries处能够找到自己的笔记，但是笔记中的插图显示不出来，说明插图可能并没有同步到Zotero账号？

**Reply by EmmaThompson123** [reply]
我也发现web library里的笔记没有我插入的截图，话说怎么打开查看*.sqlite里面保存了哪些东西

**Reply by windingwind** [reply]
again. 
1. 笔记图片可以同步
2. 同步后显示空白，是界面没刷新。重新打开笔记就有了
3. 如果用webdav同步附件，则不能在网页端看到。网页端不支持webdav

**Comment by windingwind** [comment]
不建议直接修改本地文件，会破坏同步。图片都是同步的，网页版的显示问题可向zotero社区反馈
